src/main/java/org/opentripplanner/middleware/auth/Permission.java

```diff
@@ -0,0 +1,5 @@
+package org.opentripplanner.middleware.auth;
+
+public enum Permission {
```

Comment:
Is this about permissions for admin users?

Reply:
@evansiroky, this has javadoc as of https://github.com/ibi-group/otp-middleware/pull/14/commits/4b0cb1ed711fb7ec834563340d2eafb61abba4d0